desktop/package.json

```diff
@@ -33,7 +33,7 @@
 		"test:e2e": "jest -c test/e2e/config/jest.config.js"
 	},
 	"devDependencies": {
-		"@automattic/calypso-build": "^10.0.0",
+		"@automattic/calypso-build": "^9.0.0",
```

Comment:
I guess this `10.0.0` fails because it wasn't published, right?
This is ok for now, but after the release we should bump desktop as well.

Reply:
Yep! Will do.